definitions/rest-definitions.d.ts

```diff
@@ -13,10 +13,12 @@ declare module "rest-definitions" {
 
     export interface DeploymentStatusReport {
         appVersion: string;
-        clientUniqueId: string;
-        deploymentKey: string; 
+        clientUniqueId?: string;
+        deploymentKey: string;
+        previousDeploymentKey?: string;
+        previousLabelOrAppVersion?: string;
```

Comment:
I think this makes sense if we are going to use these properties somewhere else, but for now, I can't forsee this ever being used outside of reporting deployment status updates to a new version. As such, I think I prefer to keep it flat.